pkg/cmd/printer/printer.go

```diff
@@ -37,11 +39,13 @@ const (
 )
 
 type Config struct {
-	Kind          string
-	OutPath       string
-	OutFile       io.WriteCloser
-	ContainerMode ContainerMode
-	RelativeTS    bool
+	Kind               string
+	OutPath            string
+	OutFile            io.WriteCloser
+	ContainerMode      ContainerMode
+	RelativeTS         bool
+	ForwardDestination string
```

Comment:
It may be better to use an instance of the settings structures from here directly: https://github.com/IBM/fluent-forward-go/blob/c2b759c69bc087d790145d686a8c23dd599dffd1/fluent/client/client.go#L83

This would allow us to configure any options then but needs consideration on how to pass them.